src/ui/server/helpers/whole-number-validation/index.ts

```diff
@@ -12,10 +13,12 @@ import { isNumber, numberHasDecimal, isNumberBelowMinimum } from '../number';
  * @returns {object} errors
  */
 const wholeNumberValidation = (responseBody: RequestBody, errors: object, errorMessage: string, field: string) => {
-  const hasDecimal = numberHasDecimal(Number(responseBody[field]));
-  const isBelowMinimum = isNumberBelowMinimum(Number(responseBody[field]), 0);
+  // removes commas from validation test as commas are valid
+  const numberWithoutCommas = removeCommasFromString(responseBody[field]);
+  const hasDecimal = numberHasDecimal(Number(numberWithoutCommas));
+  const isBelowMinimum = isNumberBelowMinimum(Number(numberWithoutCommas), 0);
 
-  if (!isNumber(responseBody[field]) || hasDecimal || isBelowMinimum) {
+  if (!isNumber(numberWithoutCommas) || hasDecimal || isBelowMinimum) {
```

Comment:
Can we move this to a variable above like the others, to be consistent? Looks like maybe `isNotANumber`

Also very minor, but could the comment say `// remove` without the S